help.go

```diff
@@ -226,23 +210,40 @@ func printNodeDetail(w *helpWriter, node *Node, hide bool) {
 		}
 	}
 	if w.FlagsLast {
-		printFlags()
+		PrintFlags(w, node)
+	}
+}
+
+func PrintFlags(w *HelpWriter, node *Node) {
+	if flags := node.AllFlags(true); len(flags) > 0 {
+		groupedFlags := collectFlagGroups(flags)
+		for _, group := range groupedFlags {
+			w.Print("")
+			if group.Metadata.Title != "" {
+				w.Wrap(group.Metadata.Title)
+			}
+			if group.Metadata.Description != "" {
+				w.Indent().Wrap(group.Metadata.Description)
+				w.Print("")
+			}
+			writeFlags(w.Indent(), group.Flags)
+		}
 	}
 }
 
-func writeCommandList(cmds []*Node, iw *helpWriter) {
+func writeCommandList(cmds []*Node, iw *HelpWriter) {
```

Comment:
I think if some of these functions are exposed, they probably all should be no?

Reply:
I was avoiding this to prevent too many public functions but i went ahead and added these too.